src/zenml/integrations/aws/orchestrators/sagemaker_orchestrator.py

```diff
@@ -553,8 +554,7 @@ def prepare_or_run_pipeline(
                     enabled=True,
                 )
                 next_execution = (
-                    deployment.schedule.start_time
-                    or datetime.now(timezone.utc)
+                    deployment.schedule.start_time or utc_now()
```

Comment:
@bcdurak this was `datetime.now(timezone.utc)` even without the previous PR. Can you check that this is ok